src/ir/block_builder_test.cc

```diff
@@ -130,11 +130,16 @@ TEST_F(BlockBuilderTest, AddOperationUpdatesOperationList) {
       builder.AddOperation(context_.GetOperator("core.minus"), {}, {}));
   const Operation* merge_op = std::addressof(
       builder.AddOperation(context_.GetOperator("core.merge"), {}, {}));
+  const Operation* merge_op_with_module = std::addressof(builder.AddOperation(
+      context_.GetOperator("core.merge"), {}, {}, std::make_unique<Module>()));
```

Comment:
What does an empty implementation `Module` mean when we drop this to the analysis? Does it mean that the data doesn't go through the module at all? Or does it mean the default behavior of every input going to every output?

Reply:
It is really up to the semantics of the operation. (See my other comment about the `impl_module` being a mechanism to associate lambda's with operation.)

Reply:
Sounds good.